test/integration/admin_login_test.rb

```diff
@@ -207,7 +207,7 @@ def setup
     assert page.has_content?('black nectar')
 
     visit admin_items_path
-    select 'true', from: 'retire[status]'
+    select 'true', from: "retire[status_#{@item.id}]"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.